packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -361,6 +361,7 @@ export class NotifiFrontendClient {
           authorization = newAuthorization;
         }
       } catch (e: unknown) {
```

Comment:
As far as I know, This section indicate that the token expired date will be extended while being close to the expired date of jwt. 
A question: wondering if this method is already deprecated and will ALWAYS fail, meaning it will 100% goes into `catch` block? if this is the case, why don't we just simply remove all this `refreshAuthoriztion` related logic (line 353 - 375) and also add some handling logic instead of making it bening  logged out eariler?

Reply:
The `refreshAuthorization` method still works perfectly well as of now. It's marked deprecated because we want to slowly move the SDK over to the Initial/Refresh/Normal/Elevated token pattern that I implemented as part of 2FA support in Admin Portal. Once we change _all_ clients to use the new pattern, we can safely delete `refreshAuthorization`.

Reply:
Ohoh, I see. Then it makes sense. thank you!